app/controllers/induction_programme_choices_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class InductionProgrammeChoicesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :ensure_induction_coordinator
+
+  def show
+    @programme_choices = [
+      OpenStruct.new(id: "funded_training_provider", name: "Use a training provider funded by the Department for Education"),
+      OpenStruct.new(id: "free_development_materials", name: "Use the free development materials"),
+      OpenStruct.new(id: "design_our_own", name: "Design your own induction based on the Early Career Framework"),
+      OpenStruct.new(id: "not_yet_known", name: "I don't know yet"),
+    ]
+  end
+
+  def create
+    @cohort = Cohort.find_or_create_by!(start_year: 2021)
```

Comment:
Can we have a todo here to remove the hard-coded value?

Reply:
Or do we expect to require devs to change this value yearly?

Reply:
Yes, happy to move it to a constant actually.